src/tag_parsers.rs

```diff
@@ -585,7 +662,7 @@ mod tests {
             let information_to_account_owner = (0..information_to_account_owner_count)
                 .map(|_| information_to_account_owner_text.to_string())
                 .collect::<Vec<String>>()
-                .join("\n");
+                .join("");
```

Comment:
Was this a bug before?

Reply:
I think so. Have a look at the changes in `tests/data/mt940/full/betterplace/sepa_mt9401.json` for example:
`Grund nicht s\npezifizie?22rt`
The format doesn't look like line breaks in the sta file necessarily translate to linebreaks in the output